Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM elixir:latest
+
+RUN mkdir /app
+COPY . /app
+WORKDIR /app
+
+RUN mix local.hex --force
+
+RUN mix do compile
+
+RUN mix release
+ENV MIX_ENV=prod
+CMD mix deps.get && cd assets && npm install && npm run deploy && cd ..
```

Comment:
Docker builds for Phoenix application happens as 2 stages. check: https://hexdocs.pm/phoenix/releases.html#containers